### DISCUSSION TITLE
23.10.23 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#39] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - PR 작성
    - 코드 리팩토링
        - 유틸리티 클래스 추가 (`SessionUtil`)
        - 서브모듈 등록
        - 복합키 사용을 위해 `equals()` & `hashcode()` 메서드 제정의
- 내일은 어떤 일을 하나요?
    - 테스트 코드 작성법 스터디

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - 코드 커밋 메세지 작성
    - 컨트롤러 / 서비스 테스트 작성
- 내일은 어떤 일을 하나요?
    - (서비스/컨트롤러 테스트 제외) 코드 PR
    - 플레이리스트 음악 추가

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * Streaming 기능 추가
* 내일은 어떤 일을 하나요?
  * 파일 유효성 검증 (파일이 깨진건지 아닌지 검증) 공부
  * streaming 보완

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - PR 작성(미완)
  - 비동기 사용(실패)
  - ApplicationEventPublisher test(실패)
- 내일은 어떤 일을 하나요?
  - PR 완료
  - 테스트 공부
